deb-arm/Dockerfile

```diff
@@ -74,6 +76,12 @@ RUN if [ "$DD_TARGET_ARCH" = "aarch64" ] ; then \
     fi
 RUN chmod +x /usr/local/bin/curl
 
+# CI uploader, to send junit to ci-visibility during CI tests
+RUN curl -fsSL https://github.com/DataDog/datadog-ci/releases/download/v${CI_UPLOADER_VERSION}/datadog-ci_linux-arm64 --output "/usr/local/bin/datadog-ci" && \
```

Comment:
I think you need to ensure that the arch is indeed arm64 as this Dockerfile is also used for the `deb_armhf` image

Reply:
Fixed, thanks for the warning!

Reply:
My pleasure, thanks for fixing :)